### PR TITLE
=pro check binary compatibility

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -1332,12 +1329,20 @@ object AkkaBuild extends Build {
       ProblemFilters.exclude[MissingTypesProblem]("akka.remote.ReliableDeliverySupervisor$GotUid$"),
       ProblemFilters.exclude[MissingMethodProblem]("akka.remote.ReliableDeliverySupervisor#GotUid.apply")
     )
+
+    val akkaStream = Seq(
```
Comment (drewhk) [inline project/AkkaBuild.scala]: Hmm. These will be forward ported hence the current exclude right?

File: project/AkkaBuild.scala [hunk 2]
```diff
@@ -41,6 +41,7 @@ object AkkaBuild extends Build {
   val requestedScalaVersion = System.getProperty("akka.scalaVersion", "2.10.5")
   val Seq(scalaEpoch, scalaMajor) = """(\d+)\.(\d+)\..*""".r.unapplySeq(requestedScalaVersion).get.map(_.toInt)
   val streamAndHttpVersion = "2.0.1"
+  val streamAndHttpBinCompVersion = "2.0"
```
Comment (drewhk) [inline project/AkkaBuild.scala]: Probably does not matter, but shouldn't 2.0.1 be the baseline (given that 2.0 was flawed)?

**Comment by drewhk** [comment]
LGTM apart from nitpicks

File: project/AkkaBuild.scala (hunk 2 above)
Comment (rkuhn) [inline project/AkkaBuild.scala]: Yes, I agree, and then the list of exceptions should be empty to start with.

File: project/AkkaBuild.scala (hunk 1 above)
Comment (2m) [inline project/AkkaBuild.scala]: These are incompatibilities between `2.0` and current `stream-http-2.0` (currently same as `2.0.1`) therefore these incompatibilities need to be filtered out.

File: project/AkkaBuild.scala (hunk 2 above)
Comment (2m) [inline project/AkkaBuild.scala]: I do not think this matters, as 2.0 was not flawed regarding API.

**Comment by 2m** [comment]
Changed to `2.0.1`. Failure was:

```
WebsocketServerSpec
[info]   - for echoing user handler *** FAILED *** (75 milliseconds)
[info]     java.lang.AssertionError: expected OnNext, found OnError(java.lang.IllegalArgumentException: requirement failed: Cannot pull port twice
```

**Comment by ktoso** [comment]
The pulling port twice issue is tracked here: https://github.com/akka/akka/issues/19094

**Comment by ktoso** [comment]
LGTM

**Pull request merged**